pytest_reportportal/service.py

```diff
@@ -630,14 +630,33 @@ def get_marker_value(item, keyword):
             get_marker = getattr(item, "get_closest_marker")
         except AttributeError:
             get_marker = getattr(item, "get_marker")
-        attributes = [{"value": get_marker_value(item, k)}
-                      for k in item.keywords if get_marker(k) is not None
-                      and k not in self.ignored_attributes]
-
-        attributes.extend([{"value": tag} for tag in
-                           item.session.config.getini('rp_tests_attributes')])
+        raw_attributes = [
```

Comment:
I'd split the functionality of that method into multiple methods or functions.

Reply:
I have created a new get_raw_attr function inside _get_item_markers to contain some of it's functionality